### DISCUSSION TITLE
archives/25d8

### DISCUSSION THREAD

**Discussion by giscus** [SMGoro/issues-json-generator#13] [General]
# archives/25d8

本人因为一晚上没睡着觉，闭着眼睛胡思乱想的时候突然蹦出来的想法，结果早上补觉完起来之后也一直挥之不去，所以就索性记录下来了）  本文只是本人的一时兴起，从当前的问题以及解决思路出发，不涉及任何具体的技术性问题，请各路大佬刀下留情，图一乐就好（）  引言 我们都知道，现在Minecraft的多人游戏有许多种联机方式，不过大致可以分为如下几种：  虚拟局域网/类p2p点对点连接联机（案例：hamac

https://blog.imc.re/archives/25d8

**Comment by HaoTian22** [comment]
我在想能不能在基于好友联机的场景弄去中心化的游戏，谁想玩就当一段时间的临时host，存档在结束游戏后即时同步到其他玩家电脑上（或者是用一些网盘当中转站），解决朋友之间不同游戏时间的共同整合包游玩问题
目前貌似还并没有特别好的自动化方案（可能git+启动器脚本算一个？）
但是也需要保证没有玩家对存档进行有害的操作，朋友之间完全信任
